utils.py

```diff
@@ -57,4 +80,9 @@ def get_pulpdocs_members() -> list[str]:
             "This mean we can't know if we should manage the doc-related workflows."
         )
 
-    return [line.strip()[8:] for line in response.content.decode().split("\n") if "- name:" in line]
+    repolist = yaml.safe_load(response.content.decode())
+    return [
+        repo["name"]
+        for repo in itertools.chain(*repolist["repos"].values())
+        if "subpackage_of" not in repo
+    ]
```

Comment:
That's better :sweat_smile:
What I want to change in the format is to have the `repo_type` and `subpackages` childs of repo, so `repos` would only have repo entries.